reconcile/gql_definitions/skupper_network/skupper_networks.gql

```diff
@@ -0,0 +1,98 @@
+# qenerate: plugin=pydantic_v1
+
+query SkupperNetworks {
+  skupper_networks: skupper_network_v1 {
+    identifier
+    siteConfigDefaults {
+      clusterLocal
+      console
+      consoleAuthentication
+      consoleIngress
+      controllerCpuLimit
+      controllerCpu
+      controllerMemoryLimit
+      controllerMemory
+      controllerPodAntiaffinity
+      controllerServiceAnnotations
+      edge
+      ingress
+      routerConsole
+      routerCpuLimit
+      routerCpu
+      routerMemoryLimit
+      routerMemory
+      routerLogging
+      routerPodAntiaffinity
+      routerServiceAnnotations
+      routers
+      serviceController
+      serviceSync
+      skupperSiteController
+    }
+    namespaces {
+      name
+      delete
+      skupperSite {
+        delete
+        config {
+          clusterLocal
+          console
+          consoleAuthentication
+          consoleIngress
+          controllerCpuLimit
+          controllerCpu
+          controllerMemoryLimit
+          controllerMemory
+          controllerPodAntiaffinity
+          controllerServiceAnnotations
+          edge
+          ingress
+          routerConsole
+          routerCpuLimit
+          routerCpu
+          routerMemoryLimit
+          routerMemory
+          routerLogging
+          routerPodAntiaffinity
+          routerServiceAnnotations
+          routers
+          serviceController
+          serviceSync
```

Comment:
Would it make sense to have a fragment for `SkupperConfig` this in this folder like `skupper_network/skupper_config.gql`?

Reply:
There are 2 skupper configs `SkupperSiteConfigDefaults_v1` for the skupper network, which defines the defaults for all sites, and `SkupperSiteConfig_v1`, which is basically the same, except for `skupperSiteController`, to override the defaults from the network. I don't know if that can be done via a fragment.

Reply:
Ah I understand. Makes sense :+1: